spec/octokit/client/organizations_spec.rb

```diff
@@ -156,6 +156,13 @@
       end
     end # .team_by_name
 
+    describe '.team_permission_level', :vcr do
```

Comment:
It looks like this test is failing at the moment because the VCR cassette is missing. It's fine just to use stubbing instead.